reference/profiles.rst

```diff
@@ -0,0 +1,243 @@
+.. _profiles:
+
+Profiles
+=========
+
+Profiles allows users to set a complete configurateion set for **settings**, **options**, **environment variables**, and **build
+requirements** in a file. They have this structure:
+
+.. code-block:: text
+
+    [settings]
+    setting=value
+
+    [options]
+    MyLib:shared=True
+
+    [env]
+    env_var=value
+
+    [build_requires]
+    Tool1/0.1@user/channel
+    Tool2/0.1@user/channel, Tool3/0.1@user/channel
+    *: Tool4/0.1@user/channel
+
+Profile files are useful to gather some configurations that would have to be introduced as arguments in command line instructions like
+``$ conan install`` or ``$ conan create``. Therefore, profile files ussage is highly recommended as anyone could share the whole
+configuration to build a project with settings and options, set set environment variables needed for compilation and even set requirements
+for building.
+
+.. code-block:: text
+   :caption: *myprofile*
+
+    [settings]
+    compiler=clang
+    compiler.version=3.5
+    compiler.libcxx=libstdc++11
+
+    [options]
+    MyLib:shared=True
+
+    [env]
+    CC=/usr/bin/clang
+    CXX=/usr/bin/clang++
+
+As said, you can use it instead of command line arguments:
+
+.. code-block:: bash
+
+    $ conan create demo/testing -pr=myprofile
+
+Profiles can be located in different folders, for example, the default ``<userhome>/.conan/profiles``, and be referenced by absolute or
+relative path:
+
+.. code-block:: bash
+
+    $ conan install --profile /abs/path/to/profile  # abs path
+    $ conan install --profile ./relpath/to/profile  # resolved to current dir
+    $ conan install --profile profile  # resolved to user/.conan/profiles/profile
+
+You can list and show existing profiles with the ``$ conan profile`` command:
+
+.. code-block:: bash
+
+    $ conan profile list
+    default
+    myprofile1
+    myprofile2
+    ...
+
+    $ conan profile show myprofile1
+    Configuration for profile myprofile1:
+
+    [settings]
+        os: Windows
+        arch: x86_64
+        compiler: Visual Studio
+        compiler.version: 15
+        build_type: Release
+    [options]
+    [build_requires]
+    [env]
+
+You can use ``$PROFILE_DIR`` in your profile and it will be replaced with the absolute path to the profile file.
+It is useful to declare relative folders:
+
+.. code-block:: text
+
+    [env]
+    PYTHONPATH=$PROFILE_DIR/my_python_tools
+
+.. note::
+
+    If you specify a profile in a conan command, like `conan create` or `conan install` the base profile ``~/.conan/profiles/default`` won't
+    be applied. If you want to apply it use the ``include`` directive explained later in this page.
+
+Package settings and env vars
```

Comment:
Remove all this, I think for the reference is enough to say that the profiles can be in the default profile page at ~/.conan/profiles or in any other folder. Maybe reuse this example to explain how to use a profile to point to a different compiler.